src/Workspaces/Remote/Core/BrokeredServiceDescriptors.cs

```diff
@@ -81,6 +81,8 @@ protected override JsonRpcConnection CreateConnection(JsonRpc jsonRpc)
         CreateDebuggerServiceDescriptor("SymbolLocatorService", new Version(0, 1), new MultiplexingStream.Options { ProtocolMajorVersion = 3 });
     public static readonly ServiceRpcDescriptor DebuggerSourceLinkService =
         CreateDebuggerServiceDescriptor("SourceLinkService", new Version(0, 1), new MultiplexingStream.Options { ProtocolMajorVersion = 3 });
+    public static readonly ServiceRpcDescriptor ProjectSystemQueryExecutionService =
+        CreateDescriptor(new ServiceMoniker("Microsoft.VisualStudio.ProjectSystem.Query.Remoting.QueryExecutionService", new Version(0, 2)));
```

Comment:
Follow the pattern and create a static `CreateProjectSystemServiceDescriptor` method (or something like that)?

Reply:
this seems fine as-is - since we only have one for now.  If we have more with similar properties we can extract it out.